include/wx/numformatter.h

```diff
@@ -34,6 +34,8 @@ class WXDLLIMPEXP_BASE wxNumberFormatter
     static wxString ToString(wxLongLong_t val,
                              int style = Style_WithThousandsSep);
 #endif // wxHAS_LONG_LONG_T_DIFFERENT_FROM_LONG
+    static wxString ToString(wxULongLong_t val,
+                             int style = Style_WithThousandsSep);
```

Comment:
Shouldn't this be inside the wxHAS_LONG_LONG_T_DIFFERENT_FROM_LONG prep. ?

Reply:
No because `unsigned [long] long` is always a different type, even if `long == long long`.